docs/source/python/install.rst

```diff
@@ -107,17 +107,41 @@ a custom path to the database from Python:
 Differences between conda-forge packages
 ----------------------------------------
 
-PyArrow is packaged on `conda-forge <https://conda-forge.org/>`_ as three
+On `conda-forge <https://conda-forge.org/>`_, PyArrow is published as three
 separate packages, each providing varying levels of functionality. This is in
 contrast to PyPi, where only a single PyArrow package is provided.
 
 The purpose of this split is to minimize the size of the installed package for
 most users (``pyarrow``), provide a smaller, minimal package for specialized use
 cases (``pyarrow-core``), while still providing a complete package for users who
-require it (``pyarrow-all``).
+require it (``pyarrow-all``). What was historically ``pyarrow`` on
+`conda-forge <https://conda-forge.org/>`_ is now ``pyarrow-all``, though most
+users can continue using ``pyarrow``.
 
-The table below lists the functionality provided by each package and may be
-useful when deciding to use one package over another:
+The ``pyarrow-core`` package includes the following functionality:
+
+- :ref:`data`
+- :ref:`compute` (i.e., ``pyarrow.compute``)
+- :ref:`io`
+- :ref:`ipc` (i.e., ``pyarrow.ipc``)
+- :ref:`filesystem` (HDFS, S3, GCS, etc.)
```

Comment:
If we mention that here, I think we should also say that those cloud filesystem are planned to moved out of pyarrow-core in the next release, and so you should install `pyarrow` if you want to rely on those being present

Reply:
`pyarrow.fs` itself is always available, though (with at a minimum just the LocalFileSystem)

Reply:
Good point, thanks. Done in 51f2fded2eb2f909b13edbee4808e610a1bc5c3c.